dask_conversions/done/Dask_Coastline_Extraction_Landsat.ipynb

```diff
@@ -23,12 +23,10 @@
     "\n",
```

Comment:
<p>There is something going on which is related to the number of images which are going into the final output - e.g 2011-12-13 looks quite different to in the previous version. I imagine this is something to do with the csv related bit - I will look into it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Agreed. From the original:
```Returning 595  observations as a dask array```

This PR:
```Returning 312  observations as a dask array```

Let me know if I can help investigate the missing data!

Reply:
The reason for this was purely that a large amount of no data values were being pulled in when input crs was not specified in dc.load. Meaning many no data observations were found - also the groupby solar day query was removing the influence of the actual data on the final output. So this issue has been solved - or at least understood.